.gitlab-ci.yml

```diff
@@ -40,6 +40,7 @@ workflow:
 stages:
   - build
   - test
+  - check
```

Comment:
by introducing another stage are we not delaying the work and making the pipeline run for longer?

Reply:
No, we introduced this additional stage so we could priotize the long running tasks. They will still start as early as possible but give priority to the frontend unit tests which wasn't the case before